<launch>
  <!-- We resume the logic in empty_world.launch, changing only the name of the world to be launched -->
  <include file="$(find gazebo_ros)/launch/empty_world.launch">
    <arg name="world_name" value="$(find targets_path_planning)/worlds/heightmap.world"/>
    <arg name="paused" value="false"/>
  </include>
  <node name="path_planner" pkg="targets_path_planning" type="path_planner.py" output="screen" />
  <node name="robot_navigator" pkg="targets_path_planning" type="robot_navigator.py" output="screen" />
</launch>
